graphql_compiler/query_formatting/common.py

```diff
@@ -28,6 +28,63 @@ def _raise_invalid_type_error(name, expected_python_type_name, value):
                                                          type(value).__name__))
 
 
+def _deserialize_anonymous_json_argument(expected_type, value):
+    """Deserialize argument. See docstring of deserialize_json_argument."""
+    allowed_types_for_graphql_type = {
+        GraphQLDate: (six.string_types,),
+        GraphQLDateTime: (six.string_types,),
```

Comment:
In this diff, can you make the errors for datetime objects a bit more understandable? You'll get the following error if you enter the value incorrectly.

`arrow.parser.ParserMatchError: Failed to match 'YYYY-MM-DDTHH:mm:ssZZ'`

Can you make it something that does not require a deep understanding of how datetime strings work?